tools/import_export_schedules.rb

```diff
@@ -31,6 +31,6 @@ def schedule_from_args(schedule)
   puts "Schedule #{schedule.name} exported to #{output_path}"
   File.write(output_path, exported_yaml)
 when 'import'
-  result = MiqSchedule.import(options[:import_yaml])
+  result = MiqSchedule.import(options[:import_yaml], options)
```

Comment:
see note above

```suggestion
  result = MiqSchedule.import(options[:import_yaml], options[:userid])
```

this also has the side effect of putting the parameter parsing in one spot, rather than having pieces of code that know the optimist parameters all over the code.

Reply:
unfortunately, we can't do it, because it is using generic [interface](https://github.com/ManageIQ/manageiq/blob/dbfb44cc0ae7289c084cd5a70dc46b6d6f9a45ad/app/models/mixins/yaml_import_export_mixin.rb#L31), so we need to have `options` here